### DISCUSSION TITLE
[Show and Tell] New Tool: Scaffold-OP for Superchain Dapps

### DISCUSSION THREAD

**Discussion by cpengilly** [ethereum-optimism/developers#262] [Show and tell]
### Description

We just released `Scaffold-OP` which is a fork of Scaffold-ETH2 but with some OP flavoring, such as the Superchain Faucet and additional dapp examples.

You can find the [quick start here](https://docs.optimism.io/builders/dapp-developers/quick-start) or go directly to the [`scaffold-op` repo](https://github.com/ethereum-optimism/scaffold-op) to get started. Please drop your feedback below! :heart

**Comment by i6nake9** [comment]
I am very much amazed by your work, I tried to run and deposit contracts, everything is very cool, I like how you used next.js , and such a simple method of deposit for contracts, I will continue to use superchain. I want to see and maximize! This is very cool!

**Comment by AtsKotzen** [comment]
I really liked it!!! It´s really helpfull for quickstarting!! Great job!!! Kudos!!!

**Comment by Brian496** [comment]
Am I the only one here that is getting errors trying to set this up? After inputting "yarn chain", "Error HH8: there's one or more errors in your config file:"  Example"invalid account: #0 for Network Mainnet - private key too short. ..."  with about 19 more lines with the same error.

**Reply by AtsKotzen** [reply]
Did you customized your .env file? It seems it´s something wrong in your hardhat.config file

**Reply by Brian496** [reply]
I don't think so, but I'll take a look at it. I'm somewhat uncertain since I'm new at this. I had a previous tutorial where, if I remember correctly, I did something with an .env file.

**Reply by Brian496** [reply]
I checked the.evn, made sure everything was correct. Still same error

**Reply by AtsKotzen** [reply]
What i did: 
1.Create a new wallet
2.Copy the hex private key(not the words)
3.Paste the hex private key on DEPLOYER_PRIVATE_KEY located inside packages/hardhat/.env.example
4.Rename the file .env.example to .env
5.Open the file hardhat.config.ts and make sure "localhost" is your default network (defaultNetwork: "localhost")

**Comment by Brian496** [comment]
I was confused by your step 3.  DEPLOYER_PRIVATE_KEY located inside
packages/hardhat/.env.example?
I reinstalled hardhat, created a new wallet, edited .env  and pasted
in WALLET_KEY="<YOUR_PRIVATE_KEY>" and set the hardhat.config.ts default
network to "localhost"
I even tried again pasting in a private key from a hardhat example file.
None of that made any difference. "Error HH8: There's one or more errors in
your config file:

  * Invalid account: #0 for network: mainnet - private key too short,
expected 32 bytes..." Just crazy. I even tried adding 0x at the beginning
of my private key, no difference.

**Reply by Brian496** [reply]
I just tried again using Windows Powershell  instead of the Linux terminal and I was able to run the yarn chain and yarn deploy commands  after going back through the install procedures, with only some minor error messages. So, I will call that a success. Thanks for the information just the same.

**Comment by opfocus** [comment]
amazing
